engines/scumm/resource.cpp

```diff
@@ -743,6 +744,7 @@ int ScummEngine::getResourceSize(ResType type, ResId idx) {
 }
 
 byte *ScummEngine::getResourceAddress(ResType type, ResId idx) {
+	Common::StackLock lock(_mutex);
```

Comment:
use better name for _mutex

Reply:
Ok, I will rename it to `_resourceAccessMutex`